gdbgui/backend.py

```diff
@@ -372,6 +376,7 @@ def main():
     parser.add_argument('--debug', help='The debug flag of this Flask application. '
         'Pass this flag when debugging gdbgui itself to automatically reload the server when changes are detected', action='store_true')
     parser.add_argument('-n', '--no_browser', help='By default, the browser will open with gdb gui. Pass this flag so the browser does not open.', action='store_true')
+    parser.add_argument('-x', '--command', help='Execute GDB commands from file file.', action='store')
```

Comment:
the help text is a little hard to follow since the argument name is "command". 

suggest renaming `command` to `gdb_cmd_file` and help text to `Execute GDB commands from file.`